src/SoundList.js

```diff
@@ -4,10 +4,15 @@ import { playCircleSharp, timeSharp, downloadSharp } from "ionicons/icons";
 import { IonIcon } from "@ionic/react";
 import PlayButton from "./components/PlayButton";
 
-function timeConvert(minutes) {
-  const min = Math.floor(Math.abs(minutes));
-  const secs = Math.floor((Math.abs(minutes) * 60) % 60);
-  return `${(min < 10 ? "0" : "") + min}:${secs < 10 ? "0" : ""}${secs}`;
+function formatTime(secs) {
```

Comment:
This is a great opportunity to write some unit tests by the way! Here's some boilerplate to help you get started:

```js
// src/__tests__/time.spec.js

import { formatTime } from './path/to/SoundList';

describe('format time', () => {
  it('should format time', () => {
    expect(formatTime(60)).toEqual(someVal)
  });
});
```

Our tests use [jest](jestjs.io), which is a testing library that can write unit tests for react and vanilla js! Think if it as JS's equivalent of JUnit.

Reply:
I think we should add a few unit tests for any edge cases you can identify as well as common cases.

Reply:
I added some basic test cases and covered less fringe edge cases, e.g., formatting 3600 seconds, formatting 3601 seconds, and formatting 3599 seconds.

I didn't add coverage for really extreme edge cases, like if `formatTime` receives negative numbers as a parameter.

For `-1`, it returns `"0-1:59:59"`, but `formatTime` receives floors of track durations, which should always be positive, so `formatTime` should only receive floors of positive numbers, making it always receive nonnegative inputs.

I think we would need to add error handling to `formatTime` if we want to handle more extreme edge cases like these. I'm not sure how "bulletproof" we want to make `formatTime`; it also doesn't check to make sure it receives an input that's of type `Number`.

Thank you for the boilerplate for testing! And, the simile regarding JUnit is helpful. Thanks for teaching me something new!